prisma/schema.prisma

```diff
@@ -0,0 +1,51 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("POSTGRES_PRISMA_URL")
+  directUrl = env("POSTGRES_URL_NON_POOLING")
+}
+
+enum StateType {
+  backlog
+  unstarted
+  started
+  completed
+  cancelled
+}
+
+enum AssigneeType {
+  iu
```

Comment:
can we make this more explicit for `internalUser`. `iu` is just an abbreviation we use to make it less redundant but the code should be more clear